_topics/en/adaptor-signatures.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Adaptor signatures
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Privacy Enhancements
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Adaptor signatures** are auxiliary signature data that commit to a a
+  hidden value.  When an adaptor signature is combined with a final
+  signature, it reveals the hidden value.  Alternatively, when combined
+  with the hidden value, it reveals the final signature.  The commitment
+  may be reused in other signatures without the secondary signers
+  knowing the hidden value or their own final signature.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Adaptor signatures never need to be published onchain.   To anyone
+  without a corresponding adaptor signature, a final signature looks
+  like any other digital signature, giving adaptor signatures efficiency
+  and privacy advantages over other onchain locking mechanisms such as
+  hashlocks and preimages.
+
+  ### Example
+
+  The multiple uses of adaptor signatures can be seen in a simple
```

Comment:
In other explanations I've heard (e.g. https://youtu.be/jWAhDH07bUY?t=2647), understanding the signature as "offset" by some random nonce has helped me better conceptualize how these signatures can be combined and subtracted.